tutorials/networking/high_level_multiplayer.rst

```diff
@@ -389,6 +532,30 @@ The game scene's root node should be named Game. In the script attached to it:
     func start_game():
         # All peers are ready to receive RPCs in this scene.
 
+ .. code-tab:: csharp C#
+
+    using Godot;
+
+    public partial class Game : Node3D // Or Node2D
+    {
+    	[Export] private PackedScene _mainMenuScene;
+    
+    	public override void _Ready()
+    	{
+    		// Preconfigure game.
+    		
+    		// Tell the server that this peer has loaded.
+    		Error error = RpcId(1, nameof(Lobby.PlayerLoaded));
```

Comment:
Alternatively:

```suggestion
    		Error error = GetNode<Lobby>("/root/Lobby").RpcId(1, Lobby.MethodName.PlayerLoaded);
```

This is what's suggested in the [_Singletons (Autoload)_](https://docs.godotengine.org/en/stable/tutorials/scripting/singletons_autoload.html) documentation page to access them. It's shorter and results in a single interop call, instead of 3.